tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -865,29 +857,35 @@ def test_ctrl_pod_shutdown_loop(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-40389")
-    @CTFailOn(error_handler)
-    def test_iam_bkt_cruds_during_ctrl_pod_failover(self):
+    def test_iam_bkt_cruds_during_ctrl_pod_rst(self):
         """
-        Verify IAM user and bucket operations while control pod is failing over
+        Verify IAM user and bucket operations while N-1 control pods are restarted
         """
-        LOGGER.info("STARTED: Verify IAM user and bucket operations while control pod is failing "
-                    "over")
+        LOGGER.info("STARTED: Verify IAM user and bucket operations while N-1 control pods "
+                    "are restarted")
+        LOGGER.info("Prereq: Scale replicas for control pod to %s", self.repl_num)
+        pod_name = self.node_master_list[0].get_all_pods(
+            pod_prefix=const.CONTROL_POD_NAME_PREFIX)[0]
+        resp = self.node_master_list[0].create_pod_replicas(num_replica=self.repl_num,
```

Comment:
pod replicas of only one control pod ?

Reply:
Actually by default we would have only one control pod and then we scale it for multiple replicas, but maybe yes, we may need to add something in teardown to come back to one if more than one